### PR TITLE
Docs Boilerplates & Readme Updates

### PR DESCRIPTION
**Pull request by Ehesp**
Hope this is ok, decided to put the docs into the main repo rather than a wiki since it keeps it pretty clean.

Readme files are empty, ready for documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,23 @@
...
-<a href="http://gabrielbull.github.io/react-desktop/" target="_blank">![Demo](https://rawgit.com/gabrielbull/react-desktop/master/docs/demo.svg "Demo")</a>
+> npm install react-desktop --save
+
+<a href="http://gabrielbull.github.io/react-desktop/" target="_blank">![Demo](https://rawgit.com/gabrielbull/react-desktop/master/docs/resources/demo.svg "Demo")</a>
 
 ## Contributing
 
-This library is in its infancy and was created to fulfill the lack of such libraries for tools like node-webkit or
-Electron.js. As such, everyone is welcome to contribute and add more components.
+This library has been created to provide to bring a native desktop experience to the web. It works extremely well with tools such as [node-webkit](http://nwjs.io) or [Electron.js](http://electron.atom.io)!
```
Comment (gabrielbull) [inline README.md]: I like the rephrasing, But I think we need to choose between _to provide_ or _to bring_.

Comment (Ehesp) [inline README.md]: Woops!

> This library has been created to bring a native desktop experience to the web.

Sounds better to me!

Comment (gabrielbull) [inline README.md]: Go for it!

**Comment by Ehesp** [comment]
Commited :)

**Pull request merged**